docs/release-notes/8.3.asciidoc

```diff
@@ -14,6 +87,78 @@
 [[release-notes-8.3.0]]
 == 8.3.0
 
+[discrete]
+[[known-issue-8.3.0]]
+==== Known issue
+
+8.3.0 has a bug where detection rules that were created or edited in 8.2 will stop running after you upgrade. Because of this, we advise against upgrading from 8.2 to 8.3.0. If you want to upgrade, _we highly recommend that you track your active rules beforehand so you can easily find and re-enable them after you upgrade_. Use the <<rules-api-find, Find rules API>> to capture a list of enabled rules before you upgrade.
+
+NOTE: Use curl or another HTTP tool to run {elastic-sec} API commands.
+
+Here is a curl command that you can run to find enabled rules:
+
+[source,console]
+--------------------------------------------------
+GET /api/detection_engine/rules/_find?per_page=10000&filter=alert.attributes.enabled:true
+--------------------------------------------------
+
+If you already upgraded from 8.2 to 8.3.0, detection rules affected by the bug will have stopped running with an error that is similar to the following example:
+
+[source,text]
+----
+<rule-type>:<UUID>: execution failed - security_exception: [security_exception] Reason: missing authentication credentials for REST request [/_security/user/_has_privileges], caused by: ""
+----
+
+You will need to disable, then re-enable affected rules to restore them to a functional state. These actions generate a new API key using the credentials of the user performing these actions. The rules' states are also reset.
```

Comment:
Edit to make identical the same passage in the 8.3.1 section -- ignore if they don't need to match exactly.
```suggestion
You will need to disable, then re-enable affected rules to restore them to a functional state. These actions generate new API keys using the credentials of the user performing these actions. The rules' states are also reset.
```